specification/resources/resource-manager/Microsoft.Resources/stable/2023-11-01/bicepRp.json

```diff
@@ -0,0 +1,134 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BicepRp",
```

Comment:
Why is it called bicepRP? is there a RP named Bicep?

Reply:
Changed the name to `BicepClient`